src/main/java/org/broadinstitute/hellbender/engine/spark/datasources/ReadsSparkSource.java

```diff
@@ -69,40 +81,24 @@ public ReadsSparkSource(JavaSparkContext ctx) {
      *                  use the default split size (determined by the Hadoop input format, typically the size of one HDFS block).
      * @return RDD of (SAMRecord-backed) GATKReads from the file.
      */
-    public JavaRDD<GATKRead> getParallelReads(final String readFileName, final String referencePath, final List<SimpleInterval> intervals, long splitSize) {
-        final Configuration conf = new Configuration();
+    public JavaRDD<GATKRead> getParallelReads(final String readFileName, final String referencePath, final List<SimpleInterval> intervals, final long splitSize) {
+        final Configuration conf = ctx.hadoopConfiguration();
```

Comment:
Since I see that we have a test for the partition size code, which gives me more confidence in this change, I'm going to leave it in, with the result that we use the context configuration everywhere (in this file anyway) and never create a new one.